packages/rum-recorder/src/domain/record/mutationObserver.ts

```diff
@@ -3,11 +3,11 @@ import { nodeOrAncestorsShouldBeHidden } from './privacy'
 import {
   getSerializedNodeId,
   hasSerializedNode,
+  nodeAndAncestorsHaveSerializedNode,
   NodeWithSerializedNode,
-  nodeIsIgnored,
-  nodeOrAncestorsIsIgnored,
+  transformAttribute,
 } from './serializationUtils'
-import { transformAttribute, serializeNodeWithId } from './serialize'
+import { serializeNodeWithId } from './serialize'
```

Comment:
Responsibilities:

* `serialize` is exporting functions to serialize DOM nodes
* `serializationUtils` is exporting utility functions used in various modules.

If a function is only used by `serialize` it should be in the `serialize` module, else it should be in `serializationUtils`. There is an exception, `setSeliarizedNode`, because it makes sens to collocate it with similar functions.